developer-workflow/c-api.rst

```diff
@@ -106,6 +109,24 @@ Guidelines for expanding/changing the public API:
 - Make sure the ownership rules and lifetimes of all applicable struct
   fields, arguments and return values are well defined.
 
+- Functions returning ``PyObject *`` must return a valid pointer on success
+  and ``NULL`` with an exception raised on error.
+  Most other API must return ``-1`` with an exception raised on error,
+  and ``0`` on success.
+
+- APIs with lesser and greater results must return ``0`` for the lesser result,
+  and ``1`` for the greater result.
```

Comment:
I don't undertand what "lesser" and "greater" mean here.

Reply:
There is an example, just below. Is that sufficient?